src/Command/Pull/PullCommand.php

```diff
@@ -26,6 +26,8 @@ protected function configure(): void
         $this
             ->acceptEnvironmentId()
             ->acceptSite()
+            ->acceptSiteInstanceId()
+            ->acceptCodebaseUuid()
```

Comment:
Ditto, we can't accept a codebase UUID here since we do things like checking out a branch from a specific environment / site instance